packages/ui/src/lib/alert/ErrorMessage.svelte

```diff
@@ -1,16 +1,30 @@
 <script lang="ts">
 import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+import { onMount } from 'svelte';
 import Fa from 'svelte-fa';
 
 import Tooltip from '../tooltip/Tooltip.svelte';
 
 export let error: string;
 export let icon = false;
+export let wrapMessage = false;
+let customClassWidth = '';
+let left = false;
+let top = false;
+
+onMount(() => {
+  if (wrapMessage) {
+    customClassWidth = 'w-max max-w-[650px] overflow-hidden text-ellipsis text-wrap';
+    left = true;
```

Comment:
Placing the tooltip on the left is associated with the fact we want to wrap the message. I'm not sure it will scale, as we will, at some point, want to wrap the message for elements on the left of the window (at error messages can be very long, even when started on the left of the window)

Reply:
@feloy I can change the boolean property name to be more specific, for example `wrapRightMessage` (and maybe in the future also add `wrapRightMessage`), to indicate the page placement of the message.

Reply:
@benoitf should I change the boolean property name to also indicate position on page so that we'll be able to add the same wrapping property for other positions in the future? right now it is not critical, but I do agree that it could be helpful for future changes/updates